Misc/NEWS.d/next/Library/2020-10-11-20-23-48.bpo-37449.f-t3V6.rst

```diff
@@ -0,0 +1 @@
+`ensurepip` now uses `importlib.resources.files()` traversable APIs
```

Comment:
Docs builds are failing because single backticks have a special meaning with RST.

```suggestion
``ensurepip`` now uses ``importlib.resources.files()`` traversable APIs
```